src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_IMPORT_ERROR;
+
+import java.nio.file.Path;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.storage.JsonAddressBookStorage;
+
+/**
+ * Imports data from the JSON or CSV file located at the specified path into FinBook.
+ */
+public class ImportCommand extends Command {
+
+    public static final String COMMAND_WORD = "import";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Imports data from the JSON or CSV file located at the specified path.\n"
+            + "Parameters: path to JSON or CSV file\n"
+            + "Example: " + COMMAND_WORD + " ./somewhere/addressbook.json";
```

Comment:
I feel that the example here should be consistent with our UG. 

Possible suggestion:
`"Example: " + COMMAND_WORD  + "./data.json  for .json files in the same directory as the FinBook executable"`